js/preload/preloadMain.js

```diff
@@ -121,6 +123,28 @@ function createAPI() {
         },
 
         /**
+         * Open file in default app.
+         */
+        openFile: function (id) {
```

Comment:
why do we need this in the ssf namespace.  this will expose api to JS.  i don't think needed here?